src/context/QuizContext/context/QuizContext.type.ts

```diff
@@ -0,0 +1,33 @@
+import { Quiz } from "../../../database/Quiz.type";
+
+export type InitialStateType = {
+  selectedQuiz: Quiz | null;
+  currentQuestion: number;
+};
+
+export type QuizContextType = {
+  quizState: InitialStateType;
+  quizDispatch: (action: QuizActionType) => void;
+};
+
+export type QuizActionType =
+  | {
+      type: "SET_QUIZ";
```

Comment:
Instead of strings can we use ENUMS here ?

Reply:
Thanks for the suggestion!